docs/tutorials/TOPPAS/examples.md

```diff
@@ -0,0 +1,83 @@
+Examples
+========
+
+The following sections explain the example pipelines TOPPAS comes with. Open them by selecting `File` > `Open example file`.
+All input files and parameters are already specified, so you can just hit `Pipeline` > `Run` (or press `F5`) and see what
+happens.
+
+## Profile data processing
+
+The file `peakpicker_tutorial.toppas` contains  a simple pipeline representing a common use case: starting with profile
+data, the noise is eliminated and the baseline is subtracted. Then, PeakPickerWavelet is used to find all peaks in the
+noise-filtered and baseline-reduced profile data. This workflow is also described in the section
+[Profile data processing](../TOPP/profile-data-processing.md). The individual steps are explained in more detail in the
+TOPPView tutorial: [Smoothing raw data](../TOPP/smoothing-raw-data.md), [Subtracting a baseline from a spectrum](../TOPP/subtracting-a-baseline-from-a-spectrum.md), and [Picking peaks](../TOPP/picking-peaks.md).
+
+![](../images/tutorials/toppas/TOPPAS_example_profile_data_processing.png)
+
+## Identification of E. coli peptides
+
+This section describes an example identification pipeline contained in the example directory, `Ecoli_Identification.toppas`.
+It is shipped together with a reduced example mzML file containing 139 MS2 spectra from an E. coli run on an Orbitrap
```

Comment:
What is shipped together? The pipeline? https://developers.google.com/style/pronouns#ambiguous-pronoun-references